composer/trainer/trainer.py

```diff
@@ -836,15 +876,30 @@ def __init__(
         # DDP.
 
         self._rng_state = None
+        # First try to load from ``load_path``.
```

Comment:
If `graceful_resumption` is True and `load_path` is None, can we set `load_path = f'{save_folder}/{save_latest}'`? (Maybe also emit a `log.info` for this case)